satpy/tests/reader_tests/test_modis_l1b.py

```diff
@@ -24,13 +24,14 @@
 import pytest
 from pytest_lazyfixture import lazy_fixture
 
-from satpy import available_readers, Scene
+from satpy import Scene, available_readers
+
 from ..utils import CustomScheduler, make_dataid
 from ._modis_fixtures import (
     AVAILABLE_1KM_PRODUCT_NAMES,
     AVAILABLE_HKM_PRODUCT_NAMES,
     AVAILABLE_QKM_PRODUCT_NAMES,
-    _shape_for_resolution
+    _shape_for_resolution,
```

Comment:
(When?) do the new linters require a final comma?

Reply:
I personally prefer the final comma. FYI isort is currently configured to follow "black" rules when reformatting the import code. This may explain some of the weird behavior of commas versus non-commas and where it breaks newlines.